splunk/splunk_container_template.yaml

```diff
@@ -2,21 +2,20 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: {{ .PodName}}
+  name: { { .PodName } }
```

Comment:
these changes will make this file no longer function as a proper go template. Perhaps we should change the extension for this file because its not a legit YAML file.

Reply:
added `requirements.lock` with a specific version, changed the workflow to use it instead.
I tried to use pip-tools to compile the requirements file, but currently encountered conflicts from pre-commit's requirements file and `importlib-resources<5.3,>=5.4`.